doc/usage.rst

```diff
@@ -539,6 +539,58 @@ early::
         return new Response(...);
     });
 
+View Handlers
+-------------
+
+View Handlers allow you to intercept a controller result and transform it before
+it gets returned to the kernel.
```

Comment:
You should mention that this is only when the controller result is not a response.